### PR TITLE
Polish the Azure.ResourceManager.Marketplace and prepare for release

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/marketplace/Azure.ResourceManager.Marketplace/src/autorest.md [hunk 1]
```diff
@@ -16,7 +16,62 @@ skip-csproj: true
...
 
- 
+rename-mapping:
+  AcknowledgeOfferNotificationProperties: AcknowledgeOfferNotificationContent
+  AcknowledgeOfferNotificationProperties.properties.acknowledge: IsAcknowledgeActionFlagEnabled
+  AcknowledgeOfferNotificationProperties.properties.dismiss: IsDismissActionFlagEnabled
+  AcknowledgeOfferNotificationProperties.properties.removeOffer: IsRemoveOfferActionFlagEnabled
+  Accessibility: PrivateStorePlanAccessibility
+  AdminAction: MarketplaceAdminAction
+  AdminRequestApprovalsResource: MarketplaceAdminApprovalRequest
```
Comment (Yao725) [inline sdk/marketplace/Azure.ResourceManager.Marketplace/src/autorest.md:26]: I saw a lot of resources and models are renamed and their main change is to interchange the order of `Approval` and `Request`. Why do we need to do that?

**Review by Yao725 — APPROVED** [review]

File: sdk/marketplace/Azure.ResourceManager.Marketplace/src/autorest.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/marketplace/Azure.ResourceManager.Marketplace/src/autorest.md:26]: It's bad name in swagger spec, according to description and actual operations, all these a `Request`, so made this rename changes

**Pull request merged**